common/src/main/java/org/jboss/hal/testsuite/fragment/WindowFragment.java

```diff
@@ -87,13 +89,17 @@ public WindowState clickSave() {
     }
 
     public void cancel() {
-        String label = PropUtils.get("modals.window.cancel.label");
-        clickButton(label);
+        clickCancel();
 
         waitUntilClosed();
         closed = true;
     }
 
+    public void clickCancel() {
```

Comment:
Doesn't seem as something which should be public, I would say, it should be rather private.
If it is required by cancelAndDismissReloadRequiredWindow, it should be most likely improved in order for this method not being required public